packages/fx-core/src/plugins/resource/bot/utils/common.ts

```diff
@@ -240,6 +240,13 @@ export function convertToConstValues<V extends string, T extends { [key in strin
   return Object.values(targetValues).find((itemValue) => value === itemValue);
 }
 
+export function convertToEnumValues<T extends string>(
```

Comment:
I have some question about this method. what we want to convert from and to what?

Reply:
convert string type to enum type.